cudaが見つからない

cudaがインストールしているはずなのに以下のビルドエラーが出た場合
```
/usr/bin/ld: cannot find -lcudart
/usr/bin/ld: cannot find -lcurand
collect2: error: ld returned 1 exit status
make[2]: *** [devel/lib/libdarknet_ros_lib.so] Error 1
make[1]: *** [CMakeFiles/darknet_ros_lib.dir/all] Error 2
make: *** [all] Error 2
```

cudaのパスへsymbolic linkを立てることでビルドが通れる
```
sudo ln -s /usr/local/cuda/lib64/libcudart.so /usr/lib/libcudart.so
sudo ln -s /usr/local/cuda/lib64/libcurand.so /usr/lib/libcurand.so
```

--- a/darknet_ros/CMakeLists.txt
+++ b/darknet_ros/CMakeLists.txt
@@ -134,9 +134,9 @@ if (CUDA_FOUND)
     pthread
     stdc++
     cuda
-    cudart
     cublas
-    curand
+    ${CUDA_LIBRARIES}
+    ${CUDA_curand_LIBRARY}
     ${Boost_LIBRARIES}
     ${OpenCV_LIBRARIES}
     ${catkin_LIBRARIES}

